### PR TITLE
Node/removeChild example throws an error

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

You cannot redeclare global "top" variable

I changed to the first example names

### Motivation

When someone try to use the code it throws an error

### Additional details

Firefox error:
```
Uncaught SyntaxError: redeclaration of non-configurable global property top
```

Brave error:
```
Uncaught SyntaxError: Identifier 'top' has already been declared
```

### CODE REVIEW

**Comment by Josh-Cena** [comment]
These code blocks belong to separate sections; they are not supposed to run within the same section. You can either experiment by refreshing the tab, or using Chrome, which allows redeclaring `let` between console inputs. But I don't think it's worthwhile trying to make all top-level variables distinct and think about their interactions.

**Comment by wbamberg** [comment]
> These code blocks belong to separate sections; they are not supposed to run within the same section. You can either experiment by refreshing the tab, or using Chrome, which allows redeclaring `let` between console inputs. But I don't think it's worthwhile trying to make all top-level variables distinct and think about their interactions.

I think the issue is that [`top` is a global](https://developer.mozilla.org/en-US/docs/Web/API/Window/top):

```js
let top = document.getElementById("top");
// Uncaught SyntaxError: redeclaration of non-configurable global property top
```

But I'm not sure I like one-letter variables. Perhaps instead we could use `id` and variable names like `parent` and `child`?

**Comment by Josh-Cena** [comment]
Wow, TIL. What about `topEl`?

**Comment by Graywolf9** [comment]
Yep, it is about `top` being a global, I use `d` and `d_nested` just because the first existing codeblock already uses it, but maybe just `element` and `nested` for all codeblocks could be good if you want to change in to a better names

**Comment by Josh-Cena** [comment]
Also, we use camelCase, so we definitely need to change `d_nested` to `dNested` or something better, preferably in this PR

**Comment by Graywolf9** [comment]
As the title es "removeChild" I changed the names and vars to `parent` and `child` as @wbamberg suggested, they sounds to be good options

**Review by wbamberg — APPROVED** [review]
👍 this looks good to me, thank you @Graywolf9 ! You could make those `let` declarations `const` I think?

Leaving open in case @Josh-Cena has feedback too.

File: files/en-us/web/api/node/removechild/index.md [hunk 1]
```diff
@@ -45,23 +45,23 @@ removeChild(child)
...
 
 To remove a specified element when knowing its parent node:
 
 ```js
-let d = document.getElementById("top");
-let d_nested = document.getElementById("nested");
-let throwawayNode = d.removeChild(d_nested);
+let parent = document.getElementById("parent");
+let child = document.getElementById("child");
+let throwawayNode = parent.removeChild(child);
```
Comment (Josh-Cena) [inline files/en-us/web/api/node/removechild/index.md:58]:
```suggestion
const parent = document.getElementById("parent");
const child = document.getElementById("child");
const throwawayNode = parent.removeChild(child);
```

File: files/en-us/web/api/node/removechild/index.md [hunk 2]
```diff
@@ -45,23 +45,23 @@ removeChild(child)
...
 ```
 
 To remove a specified element without having to specify its parent node:
 
 ```js
-let node = document.getElementById("nested");
+let node = document.getElementById("child");
```
Comment (Josh-Cena) [inline files/en-us/web/api/node/removechild/index.md:64]:
```suggestion
const node = document.getElementById("child");
```

**Review by Josh-Cena — COMMENTED** [review]
Apart from the potential improvement of changing `let` to `const`, this also LGTM.

(Suggestions below are non-exhaustive; please check all code blocks as there are some places I cannot comment on)

**Comment by Graywolf9** [comment]
Thanks! Already changed all `let`

**Comment by wbamberg** [comment]
Thanks for the fix, @Graywolf9 ! ❤️

**Pull request merged**